src/App.tsx

```diff
@@ -15,8 +50,14 @@ export const App: React.FC = () => {
 
       <div className="todoapp__content">
         <header className="todoapp__header">
-          {/* this buttons is active only if there are some active todos */}
-          <button type="button" className="todoapp__toggle-all active" />
+          <button
+            type="button"
+            className={classNames(
+              'todoapp__toggle-all', { active: activeTodos.length === 0 },
+            )}
+            style={{ opacity: allTodos.length === 0 ? 0 : 1 }}
```

Comment:
1. handle truth case first
2. simplify condition
```suggestion
            style={{ opacity: allTodos.length ? 1 : 0 }}
```
More advanced way: 
```suggestion
            style={{ opacity: +Boolean(allTodos.length) }}
```